### PR TITLE
OP-315 follow-up (also in preparation for OP-338)

### PR DESCRIPTION
**Pull request by mizzioisf**
Updated oh.bat
Aligned all configuration and related files

### CODE REVIEW

**Review by mwithi — APPROVED** [review]
Works for me!

But I had to:
- add `/var/log` and /`var/log/mysqld` folders
- change to `set MYSQL_DIR=mysql-5.7.30-win32`
- change to `set JAVA_DIR="jdk-11.0.9+11-jre"`

don't know why these little changes in Windows.


For missing folders, I would create them and commit them with inside only a .gitignore file, like this
```
# Ignore everything in this directory
*
# Except this file
!.gitignore
```

**Comment by mizzioisf** [comment]
Thank you for the test !

- aligned mysql version to 5.7.31 everywhere, also in Makefile (https://github.com/informatici/openhospital/pull/150)
- jdk file is: jdk-11.0.9.1+1, could you please double check ?

Regarding the paths you are definitely right, I will push later a reviewed version. 
We don't need to have or pack empty directories anymore: the scripts take care of crating everything, which is also parametric and user adjustable.
The default is now clean and simple and **common to all distros**:

/tmp
/data
data/db
data/log
data/dicom_storage

File: poh-bundle-win/oh/rsc/log4j.properties.dist [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # global logging to RollingFile (logs/ folder), available levels INFO, DEBUG, FINEST (debug++)
-log4j.rootCategory=INFO,RollingFile
+log4j.rootCategory=info,RollingFile
```
Comment (mwithi) [inline poh-bundle-win/oh/rsc/log4j.properties.dist]: Why lowercase here?

**Review by mwithi — CHANGES_REQUESTED** [review]
Also `docker-compose` is no longer needed
[Line 107](https://github.com/informatici/openhospital/pull/149/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R107)
[Line 109](https://github.com/informatici/openhospital/pull/149/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R109)

**Review by mwithi — APPROVED** [review]

**Pull request merged**